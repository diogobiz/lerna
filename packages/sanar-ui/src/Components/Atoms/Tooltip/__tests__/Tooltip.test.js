
import React from 'react'
import TestRenderer from 'react-test-renderer'

import ESTooltip from '../'

it('renders correctly', () => {
    const component = (
        <ESTooltip>
            <h1>ESTooltip</h1>
        </ESTooltip>
    )

    const tree = TestRenderer.create(component).toJSON()
    expect(tree).toMatchSnapshot()
})
