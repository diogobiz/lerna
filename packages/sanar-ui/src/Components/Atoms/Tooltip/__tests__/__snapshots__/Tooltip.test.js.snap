// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<h1
  className="es-tooltip"
  onMouseEnter={[Function]}
  onMouseLeave={[Function]}
>
  ESTooltip
</h1>
`;
